srcpkgs/qpdf/template

```diff
@@ -1,45 +1,38 @@
 # Template file for 'qpdf'
 pkgname=qpdf
-version=10.6.3
+version=10.6.3.0cmake1
 revision=1
 wrksrc="qpdf-release-qpdf-${version}"
-build_style=gnu-configure
-hostmakedepends="perl gettext tar"
-makedepends="pcre-devel libjpeg-turbo-devel"
+build_style=cmake
+hostmakedepends="perl pkg-config"
+makedepends="zlib-devel libjpeg-turbo-devel"
 short_desc="Content-Preserving PDF Transformation System"
 maintainer="Benjamín Albiñana <benalb@gmail.com>"
 license="Apache-2.0"
 homepage="https://github.com/qpdf/qpdf"
 changelog="https://raw.githubusercontent.com/qpdf/qpdf/stable/manual/release-notes.rst"
 distfiles="${homepage}/archive/release-qpdf-${version}.tar.gz"
-checksum=64bbb654ec19b6812284ef1bc40c57b091de5281f265f1c3c83dd8ccf6bedcf5
+checksum=729112f37d5eb53060cc8f5c4346e814670f147cb8498d47d010415754ce8087
 
 if [ "$XBPS_TARGET_NO_ATOMIC8" ]; then
-	makedepends+=" libatomic-devel"
-	LDFLAGS+=" -latomic"
+        makedepends+=" libatomic-devel"
+        LDFLAGS+=" -latomic"
 fi
 
```

Comment:
Need to patch out tests if CROSS_BUILD:
```
post_patch() {
        if [ "$CROSS_BUILD" ]; then
                vsed -e "/\(fmt_lld\|fmt_i64d\|fmt_i64lld\)/d" -i libqpdf/CMakeLists.txt
        fi
}
```

Reply:
No need to patch. Works too with `configure_args=-DLL_FMT=%lld -DRANDOM_DEVICE=/dev/urandom`

Reply:
Ah, shows how much I know my cmake-foo. Thanks!